build-logic/settings.gradle.kts

```diff
@@ -79,5 +79,6 @@ include("publishing")
 
 // Components used both at build time and GBT runtime, shipped in the distribution
 include("kotlin-dsl-shared-runtime")
+include("api-extractor")
```

Comment:
❓ Is this where we want to keep libraries / tools that are used both in build-logic and in production code?

Reply:
Looks that way.